src/cmd/graphql.go

```diff
@@ -9,9 +9,10 @@ import (
 	"strconv"
 	"strings"
 
+	"github.com/pkg/errors"
```

Comment:
`"github.com/pkg/errors"` is a [publicly archived](https://github.com/pkg/errors) project with a release date of 4 years ago.
If we really want the slick error wrapping feature, let's just pull the good stuff out of that repo and add it inside of `handleErr()`.
That also gets us one step closer to a "unified error message" if that's a long term goal